data-pipeline/bcreg/credssubmitter.py

```diff
@@ -45,6 +45,9 @@
 # seconds to wait for a credential response (prevents blocking forever)
 MAX_CRED_POSTING_TIMEOUT = int(os.getenv('MAX_CRED_POSTING_TIMEOUT', '240'))
 
+# seconds to wait for a credential response (prevents blocking forever)
+MAX_CRED_POSTING_TIMEOUT = int(os.getenv('MAX_CRED_POSTING_TIMEOUT', '240'))
```

Comment:
Lines 45 and 46 are duplicated on lines 48 and 49.

Reply:
done